src/components/MapLayerList/Layer.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I think layer, layer legend, and layer description can all be merged into one since they are all about the layer itself

Reply:
They are about the layer and will work better in separate components. Especially the legend since it can be displayed or not displayed and are built from the config file.